manifests/init.pp

```diff
@@ -24,6 +24,8 @@
 #
 # $group::              The Katello system user group
 #
+# $deployment_url::     The URL katello is deployed under
```

Comment:
I exposed this because puppet-lint complained otherwise but you're right that users might override this which could be bad. I'll fix this up.